cmd/kubectl-direct_csi/install.go

```diff
@@ -42,21 +43,33 @@ var (
 	installCRD       = false
 	overwriteCRD     = false
 	admissionControl = false
-	image            = "minio/direct-csi:" + Version
+	image            = "direct-csi:" + Version
```

Comment:
`quay.io/minio/direct-csi:` seems to be an invalid image (which will be chosen if `--image` is not specified) 

Can we change this to 

```
image = func() string {
     if Version == "" {
             return "direct-csi" // 'latest' tag will be chose if unspecified
     }
     return "direct-csi:" + Version
}()
 ```

Reply:
This was always the case. This will make sure they don't use dev versions of the project. Otherwise, `latest` can lead to silent failures and can be more messy than just dealing with the explicit failure during deploy.